### ISSUE TITLE
Types missing for `cypress-image-diff-js/dist/command`

### ISSUE THREAD

**Issue by karlhorky** [uktrade/cypress-image-diff#131]
When creating #130 I found that the types are missing for this `cypress-image-diff-js/dist/command` path that should be imported:

```
Could not find a declaration file for module 'cypress-image-diff-js/dist/command'. '/Users/k/p/courses/packages/learn.upleveled.io-website/node_modules/cypress-image-diff-js/dist/command.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/cypress-image-diff-js` if it exists or add a new declaration (.d.ts) file containing `declare module 'cypress-image-diff-js/dist/command';`ts(7016)
```

<img width="552" alt="Screenshot 2022-12-10 at 00 32 47" src="https://user-images.githubusercontent.com/1935696/206812224-25fd1a2c-f0b8-4410-bfaa-e2f31dd2db38.png">

Maybe these types could also be published so that #130 could also be merged?

**Comment by karlhorky** [comment]
not stale

**Comment by kapekost** [comment]
not stale

**Comment by jpalvarez** [comment]
I was able to solve this error while we wait for the release of version 2.0 in the following way:

First I created the following file cypress/support/commands-patch.js (note that the extension is .js and not .ts)

```
import compareSnapshotCommand from 'cypress-image-diff-js/dist/command';

export default compareSnapshotCommand;
```
Second I added the following in the cypress/support/commands.ts file

```
import compareSnapshotCommand from './commands-patch';

compareSnapshotCommand();
```

Finally, it is necessary to add this option to the tsconfig.ts file for this to work

```
{
  "compilerOptions": {
    ...
    "allowJs": true,
  }
}
```

**Comment by tamasmagyar** [comment]
Until v2.0.0 is release, can you try the [following Typescript guide](https://cypress.visual-image-diff.dev/getting-started/cypress-integration/typescript)?

**Comment by kien-ht** [comment]
This issue should be fixed after this pull request #195. Please upgrade to the latest version and see if the error still persists. 
I have created some [integration examples here](https://github.com/kien-ht/cypress-image-diff-html-report/tree/1aad688d5f4806be82a85c8bce1461cb0dbe4d79/examples) in case you want a reference.
